### DISCUSSION TITLE
Watch Expressions

### DISCUSSION THREAD

**Discussion by jasonLaster** [RecordReplay/devtools#5498] [Ideas]
It would be helpful to add expressions that are evaluated every time you pause. 

https://developer.mozilla.org/en-US/docs/Tools/Debugger/How_to/Set_Watch_Expressions

![image](https://user-images.githubusercontent.com/254562/155857416-f2c1efa2-d917-4093-8a6c-66f82cb7a83f.png)

> I believe Max asked for this feature as well.

**Comment by jonbell-lot23** [comment]
Similar to https://github.com/RecordReplay/devtools/discussions/5495
